L1Trigger/L1THGCal/python/egammaIdentification.py

```diff
@@ -117,11 +173,25 @@ def __init__(self, eta_min, eta_max, pt_min, pt_max):
         )
 
 egamma_identification_histomax = cms.PSet(
-        Inputs=cms.vstring('cl3d_firstlayer', 'cl3d_coreshowerlength', 'cl3d_maxlayer', 'cl3d_srrmean'),
+        Inputs=cms.vstring(input_features_histomax['v8_352']),
         CategoriesEtaMin=cms.vdouble([cat.eta_min for cat in categories]),
         CategoriesEtaMax=cms.vdouble([cat.eta_max for cat in categories]),
         CategoriesPtMin=cms.vdouble([cat.pt_min for cat in categories]),
         CategoriesPtMax=cms.vdouble([cat.pt_max for cat in categories]),
-        Weights=cms.vstring(bdt_weights_histomax),
-        WorkingPoints=cms.vdouble([wps[eff] for wps,eff in zip(working_points_histomax,tight_wp)]),
+        Weights=cms.vstring(bdt_weights_histomax['v8_352']),
+        WorkingPoints=cms.vdouble([wps[eff] for wps,eff in zip(working_points_histomax['v8_352'],tight_wp)]),
         )
+
+# Era Modification for HGCal v9: use correct training and WP
+phase2_hgcalV9.toModify(egamma_identification_histomax,
```

Comment:
The three `toModify` statements can be grouped in a single one since the same `PSet` is modified.